src/Microsoft.VisualStudio.Threading.Analyzers/Microsoft.VisualStudio.Threading.Analyzers.csproj

```diff
@@ -34,105 +28,50 @@
-    <EmbeddedResource Include="Strings.cs.resx" />
-    <EmbeddedResource Include="Strings.de.resx" />
-    <EmbeddedResource Include="Strings.es.resx" />
-    <EmbeddedResource Include="Strings.fr.resx" />
-    <EmbeddedResource Include="Strings.it.resx" />
-    <EmbeddedResource Include="Strings.ja.resx" />
-    <EmbeddedResource Include="Strings.ko.resx" />
-    <EmbeddedResource Include="Strings.pl.resx" />
-    <EmbeddedResource Include="Strings.pt-BR.resx" />
-    <EmbeddedResource Include="Strings.resx">
+    <EmbeddedResource Update="Strings.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Strings.Designer.cs</LastGenOutput>
     </EmbeddedResource>
-    <EmbeddedResource Include="Strings.ru.resx" />
-    <EmbeddedResource Include="Strings.tr.resx" />
-    <EmbeddedResource Include="Strings.zh-Hans.resx" />
-    <EmbeddedResource Include="Strings.zh-Hant.resx" />
   </ItemGroup>
   <ItemGroup>
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.cs.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.de.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.es.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.fr.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.it.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ja.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ko.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.pl.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.pt-BR.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.ru.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.tr.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.zh-Hans.xlf" />
-    <XliffResource Include="MultilingualResources\Microsoft.VisualStudio.Threading.Analyzers.zh-Hant.xlf" />
+    <Content Include="tools\*.ps1" Pack="true" PackagePath="tools\" />
+  </ItemGroup>
+  <ItemGroup>
+     <XliffResource Include="MultilingualResources\*.xlf" />
   </ItemGroup>
   <ItemDefinitionGroup>
-    <Reference>
-      <!-- Prevent all references from being packaged since VS supplies them. -->
-      <ExcludeFromNuPkg>true</ExcludeFromNuPkg>
-    </Reference>
+    <PackageReference>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
   </ItemDefinitionGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.2.0" />
```

Comment:
My default was the keep the package versions constant. But in some cases (like xunit) I was forced to upgrade to get tests running on CoreCLR which was one of the benefits we anticipated from this change. Some package dependencies are dropped because we don't need them any more (because PackageReference gives us greater control over which dependencies propagate).
We set `PrivateAssets="all"` so that the dependency is not expressed in the nuget package. We don't want this package to express any dependencies because they are analyzers. It is inconsistent here but doesn't matter because an ItemDefinitionGroup exists that sets it to all of them. So I've simplified it in a new commit that removes all the redundant metadata to reduce confusion.